sqlengine/postgres_engine_test.go

```diff
@@ -168,7 +168,7 @@ func migrationTest(connectionString, dbname string) {
 		}
 	}
 
-	usersStatement := "select usename from pg_user where usesuper != true and usename != current_user"
+	usersStatement := "select usename from pg_user where usesuper != true and usename != current_user;"
```

Comment:
Just out of curiosity: was this breaking something or done for consistency/aesthetic reasons?